backend/coins/eth/etherscan/etherscan.go

```diff
@@ -603,24 +603,33 @@ func (etherScan *EtherScan) FeeTargets(ctx context.Context) ([]*ethtypes.FeeTarg
 	if err != nil {
 		return nil, err
 	}
-	baseFeeWei := int64(baseFee * 1e9)
-	// Conversion from 10x Gwei to Wei.
+	baseFeeWei := big.NewInt(int64(baseFee * 1e9))
```

Comment:
idk if it is a problem but would `new(big.Int).Mul(big.NewInt(int64(baseFee)), factor)` be safer?

Reply:
can't hurt for consistency I suppose, I'll change it